src/styles/components/get-involved/mobilepopupCard.module.scss

```diff
@@ -0,0 +1,48 @@
+/*.multicontentcard {
+    display: flex;
+    align-items: center;
+    font-family: var(--font-inter);
+    border-radius: 12px;
+    background-color: var(--off-white, #FFFBF0);
+    //height: 100vh;
+}
+*/
+
+.layout {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    font-family: var(--font-inter);
+    border-radius: 12px;
+    background-color: var(--off-white, #FFFBF0);
+    width: 19.625rem;
+    height: 36.8125rem;
```

Comment:
try to make sure all your widths and heights are `px` instead of `rem` since `rem` only applies to font-sizes